packages/ngx-view3d/projects/ngx-view3d/src/lib/ngx-view3d.component.ts

```diff
@@ -0,0 +1,158 @@
+/**
+ * Copyright (c) 2020 NAVER Corp.
+ * egjs projects are licensed under the MIT license
+ */
+ import {
+  Component,
+  Input,
+  AfterViewInit,
+  ElementRef,
+  OnChanges,
+  Output,
+  EventEmitter,
+  OnDestroy,
+  Inject,
+  PLATFORM_ID,
+  SimpleChanges,
+  ViewChild,
+} from "@angular/core";
+import { isPlatformBrowser } from "@angular/common";
+import VanillaView3D, {
+  EVENTS,
+  DEFAULT_CLASS,
+  withMethods,
+  ReadyEvent,
+  LoadStartEvent,
+  LoadEvent,
+  LoadErrorEvent,
+  LoadFinishEvent,
+  ModelChangeEvent,
+  ResizeEvent,
+  BeforeRenderEvent,
+  RenderEvent,
+  InputStartEvent,
+  InputEndEvent,
+  CameraChangeEvent,
+  AnimationStartEvent,
+  AnimationLoopEvent,
+  AnimationFinishedEvent,
+  AnnotationFocusEvent,
+  AnnotationUnfocusEvent,
+  ARStartEvent,
+  AREndEvent,
+  ARModelPlacedEvent,
+  QuickLookTapEvent,
+  View3DOptions
+} from "@egjs/view3d";
+import { optionNames, optionInputs, setterNames } from "./const";
+
+@Component({
+  selector: "ngx-view3d, [NgxView3D]",
+  template: `
+    <canvas #canvas [ngClass]="_canvasElClass"></canvas>
+    <ng-content></ng-content>
+  `,
+  host: {
+    style: "display: block;",
+    class: DEFAULT_CLASS.WRAPPER
+  },
+  inputs: optionInputs
```

Comment:
Could this be a build?

Reply:
It can build without any issues.
It looks like this after it's built.
```js
NgxView3DComponent.decorators = [
    { type: Component, args: [{
                selector: "ngx-view3d, [NgxView3D]",
                template: `
    <canvas #canvas [ngClass]="_canvasElClass"></canvas>
    <ng-content></ng-content>
  `,
                host: {
                    style: "display: block;",
                    class: DEFAULT_CLASS.WRAPPER
                },
                inputs: optionInputs
            },] }
];
```

Reply:
It doesn't seem to matter if there is no problem with the operation in other versions.

This was not allowed in Conveyer (Angular 13).

Reply:
I'll test with Angular 13 after release then